src/schemas/json/package.json

```diff
@@ -233,9 +233,9 @@
               "$ref": "#/definitions/packageExportsEntryOrFallback",
               "description": "The module path that is resolved when the module specifier matches \"name\", shadows the \"main\" field."
             },
-            "./": {
+            "./*": {
               "$ref": "#/definitions/packageExportsEntryOrFallback",
-              "description": "The module path prefix that is resolved when the module specifier starts with \"name/\", set to \"./\" to allow external modules to import any subpath."
+              "description": "The module path prefix that is resolved when the module specifier starts with \"name/\", set to \"./*\" to allow external modules to import any subpath."
```

Comment:
There is no mention of `./` in the node's docs [here](https://nodejs.org/api/packages.html#package-entry-points) and `./*` is described as:
> As a last resort, package encapsulation can be disabled entirely by creating an export for the root of the package "./*": "./*". This exposes every file in the package at the cost of disabling the encapsulation and potential tooling benefits this provides

In addition to that, webpack docs mention [here](https://webpack.js.org/guides/package-exports/#support) that:
> (2) "./" is intentionally ignored as key.